spec/active_triples/resource_spec.rb

```diff
@@ -791,4 +791,20 @@ def bla
       end
     end
   end
+
+  describe ".apply_data_model" do
```

Comment:
I'd like to move these examples into `rdf_source_spec.rb`. I'm making moves toward properly unit testing `RDFSource` from the ground up, retaining this file as legacy/integration tests until we are confident in the isolated ones.  

A bunch of that work is done on #124, which I may try to prep for merge before this.